SR2021 kickstart details and reminder (#66)
srobo/team-emails: SR2021/2020-11-13-sr2021-kickstart-details.md

---
to: Potential team leaders + signed up teams
subject: Student Robotics Kickstart Information
---

Hello there!

We're pleased to announce some more details for the Student Robotics 2020 kickstart.

As you know, following on from the success of our [previous virtual competition](https://studentrobotics.org/news/2020-07-25-post-competition/), we will be running the competition in a simulated environment this year.

> 

Reviewer: Regardless of the outcome of the discussion about how they'll work, we should include some details here about what's happening with respect to user accounts and the forums.
If we're doing the same as normal then we should encourage the team leaders to have created their teams' accounts. Otherwise we should include info on what the plan is.

## Kickstart

> Kickstart will be streamed live on **21st November 2020 at 00:00**. Here we'll announce the game for this year, go into further details on how the competition year will work, and of course answer any questions you have.

Reviewer: At midnight? Rather you than me :trollface:

Author: Placeholder, honest.

---
to: Potential team leaders + signed up teams
subject: Student Robotics Kickstart Information
---

Hello there!

We're pleased to announce some more details for the Student Robotics 2020 kickstart.

As you know, following on from the success of our [previous virtual competition](https://studentrobotics.org/news/2020-07-25-post-competition/), we will be running the competition in a simulated environment this year.

## Kickstart

Kickstart will be streamed live on **21st November 2020 at 13:00 (UK Time)**, aiming to finish around 16:00. Here we'll announce the game for this year, go into further details on how the competition year will work, and of course answer any questions you have.

> If you've signed up for the competition, you'll receive the access details for your team. With these you can create user accounts for your competitors to give them access to the IDE and forum. We recommend creating accounts for competitors before kickstart to ensure they can get started straight away.

Reviewer: Are we doing Discord as well as or instead of the forums? I had thought instead of.
Either way: we should update this to mention Discord - both that we're using it and how we'll send out the signup details.

The simulator will be based in [webots](https://cyberbotics.com/#download). The download is large, and we recommend downloading it before the event to ensure you can get started immediately. Information on downloading our simulation will be made available on [the docs](https://studentrobotics.org/docs/competition-simulator/) during the event.

> During the event, we recommend setting up a call for you and your team to communicate and collaborate, both to share ideas and discuss potential questions.

Reviewer: I think we need more detail here as well as a clearer message that we will only be able to help the team during the microgames if a team-leader is present in the call.
I'm assuming that we've had the plan of having the team-leader be the one to invite us into their chats OK'd by @Scarzy?

Reviewer 2: It hadn't. I can't think of anything wrong with that approach at this time, it does at least ensure the team leader is in the call at the time.

Incidentally, I imagine most team leaders will be setting these up as Microsoft Teams calls, so we'd need volunteers to be able to support that.

Author: We've changed this to do the comms over discord instead, as it's more public from a volunteer perspective, and there's less organisation needed around accounts / scheduling (as you say we'd perhaps need volunteers to have O365 accounts or something!)

---
to: Potential team leaders + signed up teams
subject: Student Robotics Kickstart Information
---

Hello there!

We're pleased to announce some more details for the Student Robotics 2020 kickstart.

As you know, following on from the success of our [previous virtual competition](https://studentrobotics.org/news/2020-07-25-post-competition/), we will be running the competition in a simulated environment this year.

## Kickstart

> Kickstart will be streamed live on **21st November 2020 at 13:00 (UK Time)**, aiming to finish around 16:00. Here we'll announce the game for this year, go into further details on how the competition year will work, and of course answer any questions you have.

Reviewer: ```suggestion
Kickstart will be streamed live on **21st November 2020 at 13:00 GMT**, aiming to finish around 16:00. Here we'll announce the game for this year, go into further details on how the competition year will work, and of course answer any questions you have.
```

> The simulator will be based in [webots](https://cyberbotics.com/#download). The download is large, and we recommend downloading it before the event to ensure you can get started immediately. Information on downloading our simulation will be made available on [the docs](https://studentrobotics.org/docs/competition-simulator/) during the event.

Reviewer: The link will work, but better to have the new one:
```suggestion
The simulator will be based in [webots](https://cyberbotics.com/#download). The download is large, and we recommend downloading it before the event to ensure you can get started immediately. Information on downloading our simulation will be made available on [the docs](https://studentrobotics.org/docs/simulator/) during the event.
```

During the event, we recommend setting up a call for you and your team to communicate and collaborate, both to share ideas and discuss potential questions. Alternatively (or as well as), you can use our Discord server (see below).

After the main presentation we'll move into microgames, which are designed to help you get familiar with the simulator and robot.

Visit [our website](https://studentrobotics.org/events/sr2021/kickstart/) for more information, and a link to [the livestream](https://www.youtube.com/watch?v=cQOgo0Gh4iA).

## Discord

> If you've signed up for the competition, you'll soon receive an invite to our Discord server for your team. The invite link will be valid for `n` competitors, and should not be shared to those outside your team. If your institution has requested multiple teams, you'll receive multiple login links. We recommend sending these invites before kickstart to ensure they can get started straight away.

Reviewer: Is the `n` here going to be swapped out for a specific number per team? If so, does that mean we'll need to hand out further links if the team grows?
Perhaps just say "a small number" and the limit it to something like N+3 for N expected competitors? That also allows them to invite other team leaders too.

> If you've signed up for the competition, you'll soon receive an invite to our Discord server for your team. The invite link will be valid for `n` competitors, and should not be shared to those outside your team. If your institution has requested multiple teams, you'll receive multiple login links. We recommend sending these invites before kickstart to ensure they can get started straight away.

Reviewer: Historically we haven't opened the forums until the day of Kickstart. Are we doing something similar with the Discord? (Is that something we _can_ do if sharing the links early?)
For clarity I'm :+1: on sharing the links ASAP, this is just something for us to bear in mind.

Author: It's something we could do, but it'd look very strange, and we'd want to warn them. Personally i'm ok with it being open beforehand

---
to: Potential team leaders + signed up teams
subject: Student Robotics Kickstart Information
---

Hello there!

We're pleased to announce some more details for the Student Robotics 2020 kickstart.

As you know, following on from the success of our [previous virtual competition](https://studentrobotics.org/news/2020-07-25-post-competition/), we will be running the competition in a simulated environment this year.

## Kickstart

Kickstart will be streamed live on **21st November 2020 at 13:00 GMT**, aiming to finish around 16:00. Here we'll announce the game for this year, go into further details on how the competition year will work, and of course answer any questions you have.

> The simulator will be based in [webots](https://cyberbotics.com/#download). The download is large, and we recommend downloading it before the event to ensure you can get started immediately. Information on downloading our simulation will be made available on [the docs](https://studentrobotics.org/docs/simulator/) during the event.

Reviewer: Would be good to add something here acknowledging that this is the same simulator as the virtual competition ran in, but that users should download the latest version.

Reviewer: Perhaps
```suggestion
The simulator will be based in [Webots](https://cyberbotics.com/#download). The download is large, and we recommend downloading it before the event to ensure you can get started immediately. If you attended last year's virtual competition and still have Webots installed, you should upgrade to the latest version as there have been a number of bug fixes.

Our simulation (containing the robot and the arena) is a separate, much smaller, download which will be made available on [the docs](https://studentrobotics.org/docs/simulator/) during the event.
```

Reviewer: Oh, just spotted you'd also some wording.

Author: https://github.com/srobo/team-emails/pull/66/commits/b4284812eeb5c6ebbe78608cead8b1eb906d84f4

> The simulator will be based in [webots](https://cyberbotics.com/#download). The download is large, and we recommend downloading it before the event to ensure you can get started immediately. This is the same simulator as used for the [SR2020 virtual competition](https://studentrobotics.org/news/2020-07-25-post-competition/), but you should download the latest version. Information on downloading our simulation will be made available on [the docs](https://studentrobotics.org/docs/simulator/) during the event.

Reviewer: What does the "SR2020 virtual competition" link add?

Author: Context? Can't hurt to link out to things. Gives people more of a sense of what a virtual competition looks like

Reviewer: It's already been linked above and link styling makes that phrase stand out. If it weren't a link, would you want that phrase to stand out in that paragraph? Does it help someone merely scanning the paragraph find the information they want?

During the event, we recommend setting up a call for you and your team to communicate and collaborate, both to share ideas and discuss potential questions. Alternatively (or as well as), you can use our Discord server (see below).

> After the main presentation we'll move into microgames, which are designed to help you get familiar with the simulator and robot.

Reviewer: ```suggestion
After the main presentation we'll move into microgames, which are a series of challenges designed to help you get familiar with the simulator and robot.
```

Visit [our website](https://studentrobotics.org/events/sr2021/kickstart/) for more information, and a link to [the livestream](https://www.youtube.com/watch?v=cQOgo0Gh4iA).

## Discord

If you've signed up for the competition, you'll soon receive an invite to our Discord server for your team. The invite link will be valid for a small number of people (contact us if you run out), and should not be shared to those outside your team. If your institution has requested multiple teams, you'll receive a link for each. We recommend sending these invites before kickstart to ensure competitors can get started straight away.

> The Discord server has a number of general purpose channels, as well as dedicated channels for both your school / institution and individual teams (if you have multiple teams).

Reviewer: ```suggestion
The Discord server has a number of general purpose channels, as well as private channels for individual teams.
```

The simulator will be based in [webots](https://cyberbotics.com/#download). The download is large, and we recommend downloading it before the event to ensure you can get started immediately. Information on downloading our simulation will be made available on [the docs](https://studentrobotics.org/docs/competition-simulator/) during the event.

> During the event, we recommend setting up a call for you and your team to communicate and collaborate, both to share ideas and discuss potential questions. After the main presentation we'll move into microgames, which are designed to help you et familiar with the simulator and robot. If you'd like a blue-shirt to sit in on this call to support competitors and answer any further questions, please [let us know](mailto:teams@studentrobotics.org). Note that for safeguarding reasons, we may only be in the call whilst you are.

Reviewer: Oh, I was expecting this to be a lot more dynamic than needing to let us know ahead of time -- more that we'd jump into a call in response to specific requests.

Separately, have we had an OK from @Scarzy that this is something we can do with our current safeguarding setup? I think knowing we can do this is a blocker to telling team-leaders that we will be doing it.

Author: I've updated this to have the primarily method be discord, as it's much easier. We lose voice, but it's still low latency, and public (at least to us). It also makes sharing screenshots and files much simpler.

Reviewer 2: > you et familiar
typo

> Separately, have we had an OK from @Scarzy that this is something we can do with our current safeguarding setup? I think knowing we can do this is a blocker to telling team-leaders that we will be doing it.
This is basically no different to the video interviews we were planning to run prior to the last competition, with the added advantage that as it's targetting a whole team and as such 1-to-1's are far less likely to occur. So fundamentally, no, there's no reason why this couldn't occur.
We had said for the video interviews that it required 2x blueshirts, which is harder to manage with the dynamic approach. However, as the odds of 1-to-1 is reduced it may be possible to forego this requirement. I'll review over the next couple of days and produce specific guidance.

> Note that for safeguarding reasons, we may only be in the call whilst you are.
This is a tad vague as to who "you" is.
We could get away with it not being the team leader, so long as other adults were involved. However team leader is vastly preferable, as such I wouldn't advertise that option, just bring it up if anyone specifically asks.
